### PR TITLE
Org profile

### PR DESCRIPTION
**Pull request by binford2k**
This will create an org profile like you see at https://github.com/github

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Welcome to the Puppet community 👋
+
+![Puppet folk having a grand time at Portland's annual Pride parade.](https://raw.githubusercontent.com/puppetlabs/.github/org-profile/profile/Community-Marquee-1792x768.png)
```
Comment (mwaggett) [inline profile/README.md:3]: Looks like where you've put the image will end up at
```suggestion
![Puppet folk having a grand time at Portland's annual Pride parade.](https://raw.githubusercontent.com/puppetlabs/.github/profile/Community-Marquee-1792x768.png)
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- 🦊 https://voxpupuli.org
+- 🎙 https://dev.to/puppet
+
+Don't forget that all participation is governed by our [Code of conduct](https://pup.pt/conduct).
+
+### 🎁 Contributing to the ecosystem
+
+Community contributions are awesome and part of the way that we make sure that our tools continue to solve your problems. We want to keep it as easy as possible to participate and make contributions. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things, of course!
+
+Most of our projects will use our standard [contribution guidelines](https://github.com/puppetlabs/.github/blob/master/CONTRIBUTING.md). but you should check individual repositories for a more specific guide before submitting a PR.
```
Comment (mwaggett) [inline profile/README.md:24]: ooh should we make the `master`->`main` switch for this repo?

Comment (mwaggett) [inline profile/README.md:24]:
```suggestion
Most of our projects will use our standard [contribution guidelines](https://github.com/puppetlabs/.github/blob/master/CONTRIBUTING.md), but you should check individual repositories for a more specific guide before submitting a PR.
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### 💻 Wanna come work with us?
+
+Check out our 📜 [jobs listing](http://puppet.com/jobs) and see what strikes your fancy.
```
Comment (mwaggett) [inline profile/README.md:28]: ¯\_(ツ)_/¯ this feels more natural to me
```suggestion
Check out our 📜 [job listings](http://puppet.com/jobs) and see what strikes your fancy.
```

File: profile/README.md (hunk 2 above)
Comment (binford2k) [reply profile/README.md]: good call! I'll do that separately

**Pull request merged**